pcweb/pages/docs/__init__.py

```diff
@@ -112,6 +113,21 @@ def exec_blocks(doc, href):
 outblocks = []
 
 
+manual_titles = {
```

Comment:
We should put each title in the meta block on top of each md file instead of here.

Reply:
agreed let's do this in the future